integration/mock-client/blockchain/keeper.go

```diff
@@ -50,7 +50,7 @@ func handleEthCall(msg JsonrpcMessage) ([]JsonrpcMessage, error) {
 		return nil, err
 	}
 
-	if !strings.HasPrefix(data.Data, "0xb7d06888") {
+	if !strings.HasPrefix(data.Data, "0x07d76d31") { // checkForUpkeep()
```

Comment:
```suggestion
	if !strings.HasPrefix(data.Data, "0x07d76d31") { // checkForUpkeep(uint256,address)
```
Might as well just include the types to avoid confusion